drivers/net/ppp.c

```diff
@@ -33,8 +33,8 @@ LOG_MODULE_REGISTER(net_ppp, LOG_LEVEL);
 #include <zephyr/drivers/console/uart_mux.h>
 #include <zephyr/random/rand32.h>
 
-#include "../../subsys/net/ip/net_stats.h"
-#include "../../subsys/net/ip/net_private.h"
+#include <zephyr/../../subsys/net/ip/net_stats.h>
+#include <zephyr/../../subsys/net/ip/net_private.h>
```

Comment:
This looks questionable; i.e. this include path is relative to the source file path and is not using one of the global include paths (though, it does work for `include/zephyr` purely by chance).

Reply:
Sorry I should have seen these (used the script...). Yes these two should not be changed, I'll revert them. Thanks for catching it.